app/controllers/stores_controller.rb

```diff
@@ -1,22 +1,20 @@
 class StoresController < ApplicationController
 
-  def index
-
-
-  end
-
   def show
```

Comment:
We still need the index page. I cannot commit this pull request until that's fixed. @Nova840